project_tests.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
```

Comment:
Should we switch to hyphen in script names for elife projects as well?

Reply:
Too entrenched to be easily done :stop_sign:

Reply:
Okay, I'll leave it like it is for now.